### PR TITLE
RavenDB-21283 Update license information in About view

### PR DESCRIPTION
**Pull request by mateuszbartosik**
### Issue link
https://issues.hibernatingrhinos.com/issue/RavenDB-21283

### Additional description
Added 6.0 license limits to the About page

### Type of change
- New feature

### How risky is the change?
- Low 

### Backward compatibility
- Non breaking change

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing by Contributor
- It has been verified by manual testing

### Testing by RavenDB QA team
- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

**Review by ml054 — APPROVED** [review]

**Comment by mateuszbartosik** [comment]
Switched to a general formatter as @kalczur suggested

File: src/Raven.Studio/typescript/viewmodels/shell/about.ts [hunk 1]
```diff
@@ -171,15 +172,23 @@ class about extends viewModelBase {
...
-           if (licenseStatus) {
-               return licenseStatus[name] ? "icon-checkmark" : "icon-cancel";
-           }
-           return "icon-cancel";
-        });
+
+    formatLicenseStatusValue(
+        name: Exclude<keyof Raven.Server.Commercial.LicenseStatus, "Attributes" | "Version">
+    ): string | number {
+        const value = license.getStatusValue(name);
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell/about.ts:179]: what if license doesn't have given attribute. Will that display as infinity?

**Review by ml054 — APPROVED** [review]

File: src/Raven.Studio/typescript/viewmodels/shell/about.ts (hunk 1 above)
Comment (kalczur) [reply src/Raven.Studio/typescript/viewmodels/shell/about.ts:179]: Yes, but boolean fields are required, so there will be no problem. We expect unfilled fields only in numeric values ​​(and then it means infinity)

**Review by kalczur — APPROVED** [review]

File: src/Raven.Studio/typescript/viewmodels/shell/about.ts (hunk 1 above)
Comment (kalczur) [reply src/Raven.Studio/typescript/viewmodels/shell/about.ts:179]: Yes, but boolean fields are required, so there will be no problem. We expect unfilled fields only in numeric values ​​(and then it means infinity)

**Pull request merged**